src/cider/nrepl/middleware/reload.clj

```diff
@@ -0,0 +1,58 @@
+(ns cider.nrepl.middleware.reload
+  (:require
+   [clj-reload.core :as reload]
+   [clojure.main :refer [repl-caught]]
+   [clojure.string :as str]
+   [haystack.analyzer :as analyzer]
+   [nrepl.middleware.interruptible-eval :refer [*msg*]]
+   [nrepl.middleware.print :as print]
+   [nrepl.misc :refer [response-for]]
+   [nrepl.transport :as transport]
+   [orchard.misc :as misc]))
+
+(defn- user-reload
+  "clj-reload.core/reload from the user project.
+  Must be configured via clj-reload.core/init before being called."
+  []
+  (some-> (symbol "clj-reload.core" "reload") ;; Don't use mrandorsenized version
+          resolve))
+
+(defn- init
+  "Initialize clj-reload with dirs.
+  Only used for test, but necessary because of mranderson."
+  [dirs]
+  (reload/init {:dirs dirs}))
+
+(defn respond
+  [{:keys [transport] :as msg} response]
+  (transport/send transport (response-for msg response)))
+
+(defn- refresh-reply
```

Comment:
I guess you forgot to rename this.

Reply:
Yeap, initial copy paste then didn't look at it again :(